tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/pullquote_block.html

```diff
@@ -10,13 +10,13 @@
         <div class="pullquote__meta">
             {% if value.author_image %}
                 {% include "patterns/atoms/avatar/avatar.html" with avatar=value.author_image classes="avatar--small" %}
-            {% elif value.company_logo %}
-                {% image value.company_logo fill-110x70 format-webp class="pullquote__company-logo" %}
+            {% elif value.logo %}
+                {% image value.logo fill-110x70 format-webp class="pullquote__logo" %}
```

Comment:
I think it's better to leave the class as pullquote__company-logo for now just because this classname has already changed once recently and we're going to get issues rebasing / with merge conflicts. Let's change the classname in a follow-up pr if necessary.